### ISSUE TITLE
Hyper-parameter Searching

### ISSUE THREAD

**Issue by Flaick** [benedettaliberatori/T3AL#19]
Hello, thank you for this wonderful work. Though I have one question about how do you tune the hyper-parameters for the test-time adaptation in this codebase? I saw that you have early_stopping but I am confused which metric and is there a validation set to monitor and do the early stopping?

**Comment by wj0323i** [comment]
python src/train.py experiment=tt_<dataset_name> data=<dataset_name> model.video_path=</path/to/data/> model.split=<split> model.setting=<setting> data.nsplit=0 exp_name=<exp_name>想问一下，你这个代码怎么跑起来呀，老是报错

**Comment by benedettaliberatori** [comment]
Hi @Flaick, I apologize for the late reply and appreciate your interest in this work! 

Since we do not conduct training, there is no validation set. We perform inference on each test video sample, using only the given unlabeled video. At test-time, we compute a loss in an unsupervised manner (using pseudo-labels), and therefore we experimented with early stopping on the value of the loss. This is not included in the present code because it does not change performance results, but can be easily added to the `tt_method.py` forward.

Hope this solves your doubts!

**Comment by benedettaliberatori** [comment]
Hi @wj0323i, thank you for your interest in this work! 

You need to fill in the command you are attaching with the required information. The `< >` are merely placeholders. Please select the dataset you want to evaluate and add the data path.

Let me know if you need further help with it!
